assets/src/edit-story/masks/index.js

```diff
@@ -40,7 +40,7 @@ export const MaskTypes = {
 
 const CLIP_PATHS = {
   [MaskTypes.HEART]:
-    'M 0.5,1 C 0.5,1,0,0.7,0,0.3 A 0.25,0.25,1,1,1,0.5,0.3 A 0.25,0.25,1,1,1,1,0.3 C 1,0.7,0.5,1,0.5,1 Z',
+    'M.99834689.27724859C.98374997.1165844.87003101.00001922.7277144.00001922c-.0948137 0-.18162681.05102248-.23047608.13279699C.44883142.04998394.36557613 0 .27228183 0 .12998435 0 .01624632.1165652.00166847.27722932c-.00115382.007097-.00588463.0444451.00850059.10535296.0207321.0878518.06861968.1677608.13845102.23103404l.34838744.31615494.35436847-.31613565c.0698315-.0632926.1177191-.14318227.13845114-.23105333.0143856-.0608885.009655-.0982371.00852-.10533369z',
```

Comment:
Any way we could round these up a little?

Reply:
I think browsers respects 6 decimal points.

Reply:
Just for our own organization/file size, unless the shape really does need those exact curves/points

Reply:
These currently can't be rounded due to the viewbox (would need a much bigger viewpox otherwise). I'll try some more things.

Reply:
these cannot be rounded, due to the 1-based viewbox – otherwise the shape looks messed up :)